apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Yes, I agree with your idea of providing the information to user, about api's which are not executed bcz those api's. need particular config to be enabled.

But, Even if we disable `CONFIG_PTHREAD_MUTEX_TYPES,` `tc_libc_pthread_pthread_mutexattr_init, tc_libc_pthread_pthread_mutexattr_destroy`
api's will execute,then why the below condition check is needed.

```
#ifdef CONFIG_PTHREAD_MUTEX_TYPES
	tc_libc_pthread_pthread_mutexattr_init();
	tc_libc_pthread_pthread_mutexattr_destroy();
#else
	printf("tc_libc_pthread_pthread_mutexattr_init, tc_libc_pthread_pthread_mutexattr_destroy, will not be executed. CONFIG_PTHREAD_MUTEX_TYPES is not set");
#endif
```